vision/module/module_bounding.py

```diff
@@ -34,7 +34,8 @@ def getModuleBounds(dimensions, center, depth):
     x, y = center
     vert_res, horiz_res = dimensions
 
-    vert_angle = np.degrees(np.arctan((MODULE_HEIGHT / 2) / depth))
+    # switched height and width so it worked while testing
+    vert_angle = np.degrees(np.arctan((MODULE_WIDTH / 2) / depth))
```

Comment:
For this, I think the width and height should be flipped for the vert angle and horizontal angle since the module is taller than it is wide.